### ISSUE TITLE
Prettier 3.0.0

### ISSUE THREAD

**Issue by DarioMagniPLRM** [hosseinmd/swagger-typescript#46]
After updating prettier from 2.8.8 to 3.0.0, swagger-typescript stopped working.

This is the error message:
```txt
(node:11936) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)
services Completed
types Completed
httpRequest Completed
All Completed
node:internal/fs/utils:906
    throw new ERR_INVALID_ARG_TYPE(
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received an instance of 
Promise
    at writeFileSync (node:fs:2318:5)
    at formatFile (file:///D:/GitHub/math-engine/node_modules/.pnpm/swagger-typescript@6.4.0_@tanstack+react-query@4.29.19_axios@1.4.0_prettier@3.0.0_typescript@5.1.6/node_modules/swagger-typescript/lib/javascript/index.mjs:87:5)
    at file:///D:/GitHub/math-engine/node_modules/.pnpm/swagger-typescript@6.4.0_@tanstack+react-query@4.29.19_axios@1.4.0_prettier@3.0.0_typescript@5.1.6/node_modules/swagger-typescript/lib/javascript/index.mjs:71:21
    at Array.forEach (<anonymous>)
    at file:///D:/GitHub/math-engine/node_modules/.pnpm/swagger-typescript@6.4.0_@tanstack+react-query@4.29.19_axios@1.4.0_prettier@3.0.0_typescript@5.1.6/node_modules/swagger-typescript/lib/javascript/index.mjs:69:19
    at D:\GitHub\math-engine\node_modules\.pnpm\recursive-readdir@2.2.3\node_modules\recursive-readdir\index.js:88:20
    at FSReqCallback.oncomplete (node:fs:200:5) {
  code: 'ERR_INVALID_ARG_TYPE'
}

Node.js v20.4.0
```

.prettierrc

```json
{
  "arrowParens": "avoid",
  "printWidth": 160,
  "parser": "typescript",
  "singleQuote": true,
  "trailingComma": "none",
  "semi": false,
  "bracketSameLine": true,
  "bracketSpacing": true
}
```

After some testing I can confirm that:
- restoring prettier to version 2.8.8 the problem disappears
- NODE_TLS_REJECT_UNAUTHORIZED is irrelevant

**Comment by hosseinmd** [comment]
Fixed: v6.5.0

**Comment by DarioMagniPLRM** [comment]
Thank you!
I confirm that now is working properly!
